src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1272,4 +1272,13 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderDescription" xml:space="preserve">
+    <value>Prefer const char over const unit length string in StringBuilder.Append for more performant code</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderMessage" xml:space="preserve">
+    <value>Prefer const char over const unit length string in StringBuilder.Append for more performant code</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderTitle" xml:space="preserve">
+    <value>Prefer const char over const unit length string in StringBuilder.Append for more performant code</value>
+  </data>
```

Comment:
I believe we generally make these different, e.g.

Title = "Use StringBuilder.Append(char) when applicable"

Message = "Use StringBuilder.Append(char) instead of StringBuilder(string) when the input is a constant char."

Description = "StringBuilder.Append(char) is more efficient than StringBuilder.Append(string) even when the string is a single character.  When calling Append with a constant, prefer using a constant char rather than a constant string containing one character."